shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -138,7 +149,15 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     int replication = FileSystemShellUtils.getIntArg(cl, REPLICATION_OPTION, DEFAULT_REPLICATION);
     Set<String> workerSet = Collections.EMPTY_SET;
-    if (cl.hasOption(HOSTS_OPTION.getLongOpt())) {
+    if (cl.hasOption(HOST_FILE_OPTION.getLongOpt())) {
+      String hostFile = cl.getOptionValue(HOST_FILE_OPTION.getLongOpt()).trim();
+      try (BufferedReader reader = new BufferedReader(new FileReader(hostFile))) {
+        workerSet = new HashSet<String>();
+        for (String worker; (worker = reader.readLine()) != null; ) {
+          workerSet.add(worker);
```

Comment:
Add a check here to filter the empty string.

Reply:
> Add a check here to filter the empty string.

ok